real-estate-listing/README.md

```diff
@@ -16,6 +16,7 @@ was featured in the HubSpot.Extend() conference:
 
 This project can be used in a HubSpot account by:
 
+1. Use the schema file and sample data in `./resources` folder to create a custom object for Listing.
```

Comment:
@sejal27 this was supposed to be covered by the bullet point below: "Creating the Listing custom object schema in your portal..."  It's move verbose there to include links to the docs and local files, but I'm fine with this more concise line item... we should pick one though